sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/ServiceClientBuilderWithTimeoutExample.java

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob;
+
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpMethod;
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.util.Context;
+import com.azure.storage.common.StorageSharedKeyCredential;
+import com.azure.storage.common.policy.RequestRetryOptions;
+import com.azure.storage.common.policy.RetryPolicyType;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.time.Duration;
+import java.util.Locale;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * This example shows how to start using the Azure Storage Blob SDK for Java.
```

Comment:
This is not very descriptive. It should describe the fact that the sample details how to use request level timeouts.